File: sys/frontend/user/mobile/android/app/src/main/kotlin/app/dev/gengoka/domain/model/Comment.kt
package app.dev.gengoka.domain.model

data class Comment(
    val id: String,
    val answerId: String,
    val userId: String,
    val content: String,
    val createdAt: String
)

data class CommentWithUser(
    val id: String,
    val answerId: String,
    val userId: String,
    val content: String,
    val createdAt: String,
    val user: UserSummary
)

File: sys/frontend/user/web/src/types/comment.ts
import type { UserSummary } from './user';

export interface Comment {
  id: string;
  answer_id: string;
  user_id: string;
  content: string;
  status: string;
  created_at: string;
  updated_at: string;
}

export interface CommentWithUser extends Comment {
  user: UserSummary;
}

export interface CreateCommentRequest {
  content: string;
}
